src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -106,6 +106,11 @@
   (when (and gas gas-price)
     (money/wei->ether (.times gas gas-price))))
 
+(defn return-to-transaction [modal?]
+  (if modal?
+    (re-frame/dispatch [:navigate-to-modal :wallet-send-transaction-modal])
```

Comment:
custom navigation? something wrong, it shouldn't be possible to open modal from modal, we should discuss this with UX

Reply:
There's a good reason, let's discuss in chat.